content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -178,3 +178,41 @@ You can enable this feature with:
 {{< text bash >}}
 $ istioctl manifest apply --set values.galley.enableAnalysis=true
 {{< /text >}}
+
+### Ignoring specific analyzer messages via CLI
+
+Sometimes you might find it useful to hide or ignore analyzer messages in certain cases. For example, imagine a situation where a message is emitted about a resource you don't have permissions to update:
+
+{{< text bash >}}
+$ istioctl analyze -k --all-namespaces
+Warn [IST0102] (Namespace frod) The namespace is not enabled for Istio injection. Run 'kubectl label namespace frod istio-injection=enabled' to enable it, or 'kubectl label namespace frod istio-injection=disabled' to explicitly mark it as not needing injection
+Error: Analyzers found issues.
+See https://istio.io/docs/reference/config/analysis for more information about causes and resolutions.
+{{< /text >}}
```

Comment:
Maybe this should be plain rather than bash. Only the first line is syntactically bash, we don't want weird highlights showing up in the rest. (Maybe none do anyway?)

Same for the other usages below.

Reply:
👍 I marked them all as plain since there's no real bash going on here - just arguments.